README.md

```diff
@@ -281,4 +279,14 @@ If **you are not** using Docker copy `contrib/.env.sample` as `.env` in the proj
 ##### Google settings
 
 * `GOOGLE_ANALYTICS` (_str_) Google Analytics tracking code (e.g. `UA-123456-7`)
-* `GOOGLE_STREET_VIEW_API_KEY` (_str_) Google Street View Image API key
\ No newline at end of file
+* `GOOGLE_STREET_VIEW_API_KEY` (_str_) Google Street View Image API key
+
+##### Twitter settings
```

Comment:
Would be nice to document the `tweets` command just like we have documented the ones that load data into the database.